libethereum/BlockChain.cpp

```diff
@@ -194,71 +194,80 @@ void BlockChain::init(ChainParams const& _p)
 
 unsigned BlockChain::open(fs::path const& _path, WithExisting _we)
 {
-    fs::path path = _path.empty() ? db::databasePath() : _path;
-    fs::path chainPath = path / fs::path(toHex(m_genesisHash.ref().cropped(0, 4)));
-    fs::path extrasPath = chainPath / fs::path(toString(c_databaseVersion));
+    fs::path const path = _path.empty() ? db::databasePath() : _path;
+    fs::path const chainPath = path / fs::path(toHex(m_genesisHash.ref().cropped(0, 4)));
+    fs::path const chainSubPathBlocks = chainPath / fs::path("blocks");
+    fs::path const extrasPath = chainPath / fs::path(toString(c_databaseVersion));
+    fs::path const extrasSubPathExtras = extrasPath / fs::path("extras");
```

Comment:
I'd prefer these variable declarations to remain closer to where they're used (in inner scopes), some of them are only for the hypothetic case of DB laypout upgrade, which never happens now.